contents/blog/is-google-analytics-illegal-microsite.md

```diff
@@ -0,0 +1,142 @@
+---
+date: 2022-03-01
+title: "Use of Google Analytics was deemed illegal in some EU countries. We built a microsite in 48 hours to capitalize on the news."
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/is-ga-illegal-microsite.png
+author: ["cory-watilo"]
+categories: ["Inside PostHog"]
+---
+
+On January 14, 2022, the world learned that Austria declared Google Analytics to be illegal (in its default form), since it transmits IP addresses to the United States. Both Austria and The Netherlands both announced the news within days of each other.
```

Comment:
```suggestion
On January 14, 2022, the world learned that Austria had declared Google Analytics to be illegal (in its default form), since it transmits IP addresses to the United States. The Netherlands declared the same view only days later.
```